cms/envs/common.py

```diff
@@ -2530,6 +2530,15 @@
 EXAMS_SERVICE_URL = 'http://localhost:18740/api/v1'
 EXAMS_SERVICE_USERNAME = 'edx_exams_worker'
 
+############## Settings for JWT token handling ##############
+TOKEN_SIGNING = {
```

Comment:
Did you add this here because the new tests you added are running in both the LMS and CMS environments?  If so, I think that's not necessary and we can just run them in one of the two shards.  So we could remove this and the block in `cms/envs/test.py` and then use the `@skip_unless_lms` decorator to skip these tests in the CMS context.